Starls.Assets/Starls.Assets.Presentation/wwwroot/css/app.css

```diff
@@ -101,3 +101,7 @@ a, .btn-link {
 code {
     color: #c02d76;
 }
+.card-img-top{
+    max-height:350px;
```

Comment:
Essa classe parece não estar sendo utilizada, é necessaria?

Reply:
Se for pra manter isso, diria pra criar um arquivo proprio .css para pagina que quer aplica-lo, para evitar colisões.

Por exemplo. Criar o arquivo Planets.razor.css na pasta Pages e o VS vai agrupar a pagina especifica